### DISCUSSION TITLE
Discussion: Incorrect language analysis for github repository

### DISCUSSION THREAD

**Discussion by yd-yin** [github-linguist/linguist#5884] [Classification] [answered]
For this repo: https://github.com/yd-yin/FisherMatch/

The main language is Python, while Python is totally missing in the statistics, and the main detected language is HTML.

Just to mention that I do have another branch for github-page, where the main language is HTML.

Thanks a lot!

**Comment by lildude** [comment, marked as answer]
It looks like you may have changed primary branches since your last push as your sidebar analysis looks like its from your pages branch. Push another change to your main branch to force a re-analysis and all should be good again.

**Reply by yd-yin** [reply]
Perfectly solved. Thanks a lot!
